openmetadata-ui/src/main/resources/ui/src/styles/x-master.css

```diff
@@ -1004,7 +1004,7 @@ code {
 }
 
 .ant-card-body {
-  padding: 14px;
+  padding: 14px !important;
```

Comment:
Why giving important CSS?

Reply:
it is taking priority for libraries class and not taking this class